lib/ranked-model/ranker.rb

```diff
@@ -229,11 +252,13 @@ def rebalance_ranks
           gaps = current_order.size + 1
           range = (RankedModel::MAX_RANK_VALUE - RankedModel::MIN_RANK_VALUE).to_f
           gap_size = (range / gaps).ceil
+          gap_size = ranker.preferred_spread if ranker.preferred_spread && gap_size > ranker.preferred_spread
+          base_position = ranker.preferred_spread ? -gap_size * ((current_order.count + 1) / 2).ceil : RankedModel::MIN_RANK_VALUE
```

Comment:
Also a quick explainer of this would be helpful. I'm in a bit of a rush so it'd save me some figuring out time :D